src/components/VoteButton.jsx

```diff
@@ -0,0 +1,54 @@
+import { useState } from 'react'
+import { voteReview } from '../api.js'
+
+export function VoteButton({ reviewID, setReview, setDisplayAlert }) {
+  function vote(incVote) {
+    const req = { inc_votes: incVote }
+    setReview((review) => {
+      // optimistic rendering
+      const updatedReview = { ...review }
+      updatedReview.votes += incVote
+      return updatedReview
+    })
+    voteReview(reviewID, req)
+      .then(() => {
+        return
+      })
```

Comment:
If there's nothing you want it to do after the api call, then you can omit this `.then()` block and just keep the `.catch()`